packages/agent/app/create-harness.ts

```diff
@@ -16,7 +16,12 @@ export function* createHarness() {
     if(message.type === 'run') {
       let manifest: TestImplementation = yield loadManifest(message.manifestUrl);
 
-      yield runTest(parentFrame, manifest, message.path.slice(1))
+      try {
+        parentFrame.send({ type: 'lane:begin' });
```

Comment:
Might be worth sending along the path of the lane we are running here?

Reply:
Added.... indeed, it wouldn't be that useful of an event without it, would it 😆